spec/controllers/estimates_controller_spec.rb

```diff
@@ -3,10 +3,10 @@
 RSpec.describe EstimatesController, type: :controller do
   render_views
 
-  let!(:user) { FactoryBot.create(:user) }
-  let!(:project) { FactoryBot.create(:project) }
-  let!(:story) { FactoryBot.create(:story, project: project) }
-  let!(:estimate) { FactoryBot.create(:estimate, story: story, user: user) }
+  let(:user) { FactoryBot.create(:user) }
+  let(:project) { FactoryBot.create(:project) }
+  let(:story) { FactoryBot.create(:story, project: project) }
+  let(:estimate) { FactoryBot.create(:estimate, story: story, user: user) }
```

Comment:
I removed the `!` since some of the tests require the estimate to not exist now that we have the uniquness validation